django_project/realtime/tasks/headless/inasafe_wrapper.py

```diff
@@ -12,6 +12,9 @@
 LOGGER = logging.getLogger('InaSAFE Headless')
 
 
+RESULT_SUCCESS = 0
```

Comment:
Add this for more verbose conditionals. Please use it @ismailsunni for other hazards and provide another constants for other status if necessarry.